packages/design-tokens/index.ts

```diff
@@ -4,5 +4,8 @@ export * from "./src/types"
 export * from "./src/ThemeManager"
 export * from "./react"
 
-// Utils are namespaced because they are potentially useful but unrelated to direct usage of design tokens.
-export * as Utils from "./src/utils"
+export {
+  makeCSSVariableTheme,
+  mapLeafsOfObject,
+  getCSSVariableMapOfTheme,
+} from "./src/utils"
```

Comment:
Exposing everything in utils prevented us from making minor internal breaking changes. I believe these 3 are the only ones that really matter.